src/js/update.js

```diff
@@ -49,6 +48,9 @@ var update = (function() {
             giphy: {
               url: "https://giphy.com/search/"
             },
+            bing: {
+              url: "https://www.bing.com/search?q="
+            },
```

Comment:
Do we need to add the `bing` object if it is added on line 597 in the `versionUpdates["3.27.0"]` function?

Reply:
Also, I don't think changing previous update functions. Would rather reflect the change in the `state` object with a new update function.